cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -634,3 +634,47 @@ def _revert():
 
     view = navigate_to(ansible_service, "Details")
     assert view.provisioning.results.get_text_of("Status") == "successful"
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[BZ(1460788)])
+def test_service_ansible_verbosity(
+    ansible_catalog_item,
+    ansible_service_catalog,
+    ansible_service_request,
+    ansible_service,
+):
+    """Check if the different Verbosity levels can be applied to service and
+    monitor the std out
+
+    Bugzilla:
+        1460788
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: medium
+        initialEstimate: 1/6h
+        tags: ansible_embed
+    """
+    with update(ansible_catalog_item):
+        ansible_catalog_item.provisioning = {"verbosity": "3 (Debug)"}
+        ansible_catalog_item.retirement = {"verbosity": "3 (Debug)"}
+
+    ansible_service_catalog.order()
+    ansible_service_request.wait_for_request()
+    view = navigate_to(ansible_service, "Details")
+    # '3' denotes Debug level verbosity which is passed in the catalog creation.
+    assert "3" == view.provisioning.details.get_text_of("Verbosity")
+    assert "3" == view.retirement.details.get_text_of("Verbosity")
+    # Resetting verbosity value back to '0'
```

Comment:
This is an assumption based on current settings - I would re-work the test a bit to actually assert this default expected configuration value, if we're going to consider it a default, before changing it to `3`

Reply:
Yes, @mshriver I agree with you, keeping `0 (Normal)` as a default and resetting back in finalizer.
Also, validating verbosity pattern in `evm.log` would give more information that `Verbosity` set in catalog item is being passed to execute playbook and based on that we are `Verbosity` value in the ordered `Service`.